tasks/task-arguments.js

```diff
@@ -23,13 +23,13 @@ const destination = argv.destination || (destinations
   .filter(({ task }) => tasks.includes(task))[0]?.destination ?? 'public')
 
 const rootPath = dirname(__dirname)
-const destPath = resolve(rootPath, destination)
+const destPath = resolve(rootPath, slash(destination))
```

Comment:
For the `--destination` flag we have tests for both forward slashes and backslashes

We're using `slash()` to remove Windows escaped paths `\\` then `resolve()` to normalise to the platform default—Windows paths for Windows, POSIX paths for macOS/Linux

https://nodejs.org/api/path.html#pathresolvepaths
>**The resulting path is normalized** and trailing slashes are removed unless the path is resolved to the root directory.